CHANGES.md

```diff
@@ -44,6 +44,7 @@ To be released.
     Because the `Swarm<T>` constructor takes it instead.  [[#324]]
  -  `Swarm<T>` does not implement `ICollection<Peer>` anymore.  [[#326]]
  -  Added `IStore.DeleteNamespace()` method. [[#329]]
+ -  Removed `BlockChain<T>` constructor that receives `id`.  [[#TBD]]
```

Comment:
How about this?

> Removed the `id` parameter from the `BlockChain<T>` constructor, and it became to automatically detect an appropriate `BlockChain<T>.Id`.

Reply:
I wrote mention about `BlockChain<T>.Id` in 'Behavioral changes' section. is better writing them here?

Reply:
As two changes are related each other, writing a single unified log seems better to me.